templates/default/InstallChocolatey.ps1.erb

```diff
@@ -0,0 +1,147 @@
+# =====================================================================
+# Copyright 2011 - Present RealDimensions Software, LLC, and the
+# original authors/contributors from ChocolateyGallery
+# at https://github.com/chocolatey/chocolatey.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =====================================================================
+
+# Environment Variables, specified as $env:NAME in PowerShell.exe and %NAME% in cmd.exe.
+# For explicit proxy, please set $env:chocolateyProxyLocation and optionally $env:chocolateyProxyUser and $env:chocolateyProxyPassword
+# For an explicit version of Chocolatey, please set $env:chocolateyVersion = 'versionnumber'
+# To target a different url for chocolatey.nupkg, please set $env:chocolateyDownloadUrl = 'full url to nupkg file' 
+# NOTE: $env:chocolateyDownloadUrl does not work with $env:chocolateyVersion.
+# To use built-in compression (no 7zip download), please set $env:chocolateyUseWindowsCompression = 'true'
+
+$url = "https://chocolatey.org/api/v2/package/chocolatey/"
```

Comment:
I do think this should be the default url but we should allow an attribute to drive this in case users want to set to a previous version or a prerelease.